app/src/main/res/values-el/strings.xml

```diff
@@ -0,0 +1,51 @@
+    <string name="main_menu_video_setup">Εγκατάσταση βίντεο</string>
+    <string name="load_game_from_file_explanation">Μπορείτε να κατεβάσετε ROMs από παιχνίδια του Virtual Boy, και να τα φορτώσετε εδώ για να τα παίξετε.\nΜπορείτε να βρείτε δωρεάν ROMs από το Planet Virtual Boy, στη διεύθυνση https://virtual-boy.com/homebrew/.</string>
+    <string name="load_game_recent_games">Προηγούμενα παιχνίδια</string>
+    <string name="load_game_no_recent_games">Δεν έχετε παίξει ακόμα κανένα παιχνίδι. Δοκιμάστε ένα!</string>
+    <string name="game_menu_resume_game">Συνεχίστε το παιχνίδι</string>
+    <string name="game_menu_save_state">Κατάσταση αποθήκευσης</string>
+    <string name="game_menu_load_state">Κατάσταση φόρτωσης</string>
+    <string name="game_menu_state_slot">Κατάσταση τρέχουσας κατάστασης</string>
+    <string name="onscreen_input_menu_toggle_controls">Προσθήκη κουμπιών ενεργοποίησης σε λειτουργία εναλλαγής</string>
+    <string name="onscreen_input_menu_haptic_feedback">Ενεργοποίηση της απτικής ανάδρασης</string>
+    <string name="onscreen_input_menu_show_bounds">Εμφάνιση ορίων ελέγχου</string>
+    <string name="onscreen_input_menu_preview">Προεπισκόπηση</string>
+    <string name="controller_menu_manage">Διαχείριση ελεγκτών</string>
+    <string name="controller_menu_choose_controller">Επιλογή ελεγκτή για χαρτογράφηση</string>
+    <string name="controller_menu_no_controllers_detected">Δεν εντοπίστηκαν ελεγκτές...</string>
+    <string name="controller_menu_new_controller">Νέος ελεγκτής</string>
+    <string name="controller_menu_delete_controller">Διαγραφή ελεγκτή</string>
+    <string name="controller_menu_name">Όνομα ελεγκτή?</string>
+    <string name="controller_menu_create">Δημιουργία</string>
+    <string name="controller_menu_choose_rename">Επιλογή ελεγκτή για μετονομασία...</string>
+    <string name="controller_menu_choose_delete">Επιλογή ελεγκτή για διαγραφή...</string>
+    <string name="controller_menu_default_name">Ελεγκτής %1$d</string>
+    <string name="input_menu_put_mapping">Πιέστε οποιοδήποτε στοιχείο ελέγχου για να το αντιστοιχίσετε σε αυτή την είσοδο. Πατήστε παρατεταμένα για να αντιστοιχίσετε πολλαπλά στοιχεία ελέγχου.</string>
+    <string name="input_menu_add_mapping">Πιέστε οποιοδήποτε στοιχείο ελέγχου για να προσθέσετε άλλη αντιστοίχιση για αυτή την είσοδο.</string>
+    <string name="video_menu_screen_zoom_percent">Ζουμ οθόνης (ποσοστό)</string>
+    <string name="video_menu_horizontal_offset">Οριζόντια μετατόπιση</string>
+    <string name="video_menu_mode">Μορφή εμφάνισης</string>
+    <string name="video_mode_anaglyph_summary">Anaglyph (δύο χρωμάτων)</string>
+    <string name="video_mode_anaglyph_description">Δουλεύει με τα περισσότερα ζευγάρια γυαλιών 3D</string>
+    <string name="video_mode_cardboard_description">Εργάζεται με το Google Cardboard ή το Daydream</string>
+    <string name="video_mode_mono_left_summary">Μόνο για το αριστερό μάτι</string>
```

Comment:
If there's a `video_mode_mono_left_summary`, shouldn't there be a `video_mode_mono_right_summary`?